publishable/config/api.php

```diff
@@ -49,4 +49,15 @@
 
     // 'route_prefix' => 'app'
 
+    /*
+    |--------------------------------------------------------------------------
+    | Routing manifest file
+    |--------------------------------------------------------------------------
+    |
+    | Here is the path for the routing manifest that will cache the resolution
+    | strategy for each Resource in every version.
+     */
+
+    'routing_manifest_path' => storage_path('api-resources/manifest.json'),
```

Comment:
Could this be in the Laravel's framework cache? `bootstrap/cache`

![image](https://user-images.githubusercontent.com/2080215/182597471-e0592a70-98eb-4598-98fa-b4c4ff6ffbda.png)

Reply:
makes sense

Reply:
0e7adbd7ba8a134159d065a3658de2bdda68ded2